components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -0,0 +1,86 @@
+import Button from "components/atoms/Button/button";
+import { ChangeEvent, useEffect, useRef, useState, MouseEvent } from "react";
+
+const HighlightInputForm = (): JSX.Element => {
+  const [isDivFocused, setIsDivFocused] = useState(false);
+  const textAreaRef = useRef<HTMLTextAreaElement>(null);
+  const [bodyText, setBodyText] = useState<string>("");
+  const [row, setRow] = useState<number>(1);
+  const [title, setTitle] = useState<string>("");
```

Comment:
nit: remove explicit types

```suggestion
  const [bodyText, setBodyText] = useState("");
  const [row, setRow] = useState(1);
  const [title, setTitle] = useState("");
```